src/utils/misc.py

```diff
@@ -7,10 +7,12 @@
 import requests
 
 
-def send_discord(item_name, url:str = None):
+def send_discord(item_name,custom_discord_name, url:str = None):
     if url is None:
         url = "https://discord.com/api/webhooks/908071105372250213/puaS6gIYqYxTE-TBLAIs6_Qb6ZUwuygSeQfTQkpuXrSag5DPeV2gk0SctOjPy5qMHGeh"
-    data = {"content": f"Botty just found: {item_name}"}
+    data = {"content": f"Botty just found: {item_name}"
+            "username": "{custom_discord_name}"
+           }
```

Comment:
Please format nicely. One line is good enough here.
Also, how do you like the idea to type "Botty" in case no custom_discord_name is selected. And instead replace it with the custom name?